### ISSUE TITLE
Problems using the mouse via Kiwi Browser 

### ISSUE THREAD

**Issue by Katch999** [idolize/xcloud-keyboard-mouse#8]
![Screenshot_20211205-145358_Kiwi Browser](https://user-images.githubusercontent.com/95585172/144757691-1bff6e9e-deba-466c-8526-e6aa3040ea07.jpg)
![Screenshot_20211205-145402_Kiwi Browser](https://user-images.githubusercontent.com/95585172/144757692-1d6cd8b5-5b0a-48f3-985c-f32c4c6bf605.jpg)
When I try to use the extension through the kiwi browser I can't click on this second option so I can't activate the mouse  :(

**Comment by Jafetter** [comment]
I have the same problem here.

**Comment by Jafetter** [comment]
After the update the keyboard no longer connects.

**Comment by Jafetter** [comment]
It started working again, I believe that for the mouse to work on android you will have to use the same code as in the app "Panda and octopus" which are mobilizers to use the mouse and keyboards in android games. I'm going to make a donation to help you friend you've done something amazing for the community.

**Comment by Jafetter** [comment]
Boa tarde, quando configuro um preset para determinado jogo ele não salva.

**Comment by unBROken962** [comment]
I have the same problem, I searched about solves but nobody says a solve for this issue
Not even a clear answer for it

**Comment by idolize** [comment]
I don't have Kiwi browser or Android, so I have no way of debugging or testing on that kind of device.

As a result, I will not be personally supporting it. But this is open source so feel free to find a solution yourself and submit a pull request.

**Comment by Oxdf22** [comment]
I'm having the same problem and I can't use the mouse normally in kiwi Browser with mouse adapter on mobile

**Comment by rfxavier** [comment]
Yes same here kiwi browser seems to be the only option for android that accepts chrome extensions. tested on yandex, not compatible. It would really be nice to make xcloud keyboard mouse extension compatible with kiwi browser.  @idolize

**Comment by Tunta1986** [comment]
https://github.com/idolize/xcloud-keyboard-mouse/issues/8#issuecomment-990657667
Can you help me. How did you fix it?
